src/outpack/util.py

```diff
@@ -73,6 +74,12 @@ def assert_file_exists(path, *, workdir=None, name="File"):
         raise Exception(msg)
 
 
+def assert_relative_path(path: str, name: str):
+    if os.path.isabs(path):
```

Comment:
Probably we should exclude `..` from within the path too?

Reply:
Done.